google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -1747,6 +1747,22 @@ def default_acl
         # @param [Boolean] force [Deprecated] Force the latest policy to be
         #   retrieved from the Storage service when `true`. Deprecated because
         #   the latest policy is now always retrieved. The default is `nil`.
+        # @attr [Integer] requested_policy_version The requested syntax schema version of
+        #   the policy. Optional. If `1`, `nil` or not provided, `Policy#roles` and related
+        #   helpers will be accessible, but attempts to call `Policy#bindings` will raise
+        #   a runtime error. If `3` is provided, `Policy#bindings` will be accessible,
+        #   but attempts to call `Policy#roles` and related helpers will raise a runtime
+        #   error. A higher version indicates that the policy contains role
```

Comment:
This needs to be updated to reflect the subclasses. e.g. "If `1`, `nil`, or not provided, a {Google::Cloud::Storage::PolicyV1} object is returned, which provides {Google::Cloud::Storage::PolicyV1#roles} and related helpers but does not provide a `bindings` method." etc.

Reply:
Fixed.